src/GoogleFitOnFhir/Clients/GoogleFit/Handlers/GoogleFitPublishingHandler.cs

```diff
@@ -0,0 +1,66 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using EnsureThat;
+using GoogleFitOnFhir.Common;
+using GoogleFitOnFhir.Models;
+using GoogleFitOnFhir.Services;
+using Microsoft.Extensions.Logging;
+using Microsoft.Health.Common.Handler;
+
+namespace GoogleFitOnFhir.Clients.GoogleFit.Handlers
+{
+    public class GoogleFitPublishingHandler : IResponsibilityHandler<PublishRequest, Task>
```

Comment:
I hate to do this to you, but I think the term "publish" a is misnomer. I think all these related classes should be named "DataImport" or some variant. It seems confusing that the classes named publish use an importer class. Also, all the documnetation across every repository talks about "importing" data to a FHIR Service.

Reply:
No biggie.  The Azure function is named "PublishData", which is why I went with publish for a naming convention.  Easy enough to switch to "import".